src/pulselistener/pulselistener/hook.py

```diff
@@ -3,8 +3,8 @@
 
 import structlog
 
-from cli_common.pulse import create_consumer
```

Comment:
Do we have other users of cli_commont.pulse? We could remove it if it's no longer used by anyone else (it was probably always only used by pulselistener)
Same with other libraries in cli_common, maybe pulselistener is now the only remaining user.

Reply:
Unfortunately, no, shipit_api & tooltool_api are using these functions.